### ISSUE TITLE
onclick Event Handlers with `nanohtml/raw`

### ISSUE THREAD

**Issue by unitof** [choojs/nanohtml#131]
Normally, `nanohtml` intercepts the `onclick` property of its elements, and converts them to JS Event Listeners.

However, when I pass a string containing an HTML element with an `onclick` attribute through `nanohtml/raw`, the `onclick` attribute is passed through to the DOM as is, with no conversion to an Event Handler.

I have a script which parses my app's content for certain terms which should be clickable, and wraps them in a tag. That's why I'm using the `raw` function. Is it possible to pass raw HTML to `nanohtml`, but still have `nanohtml` handle `onclick`s as it usually does?

**Comment by yoshuawuyts** [comment]
@unitof could you perhaps create a code example? It's a bit hard to parse what you mean from just the text alone. Thanks!
